pages/common/mosquitto.md

```diff
@@ -0,0 +1,20 @@
+# mosquitto
+
+> An MQTT broker.
+> More information: <https://mosquitto.org/>.
+
+- Start mosquitto:
+
+`mosquitto`
+
+- Specify a configuration file to use:
+
+`mosquitto --config-file {{path/to/file.conf}}`
+
+- Listen on a specific port:
+
+`mosquitto --port {{port_number}}`
```

Comment:
Is it better to give a concrete value here ?

Reply:
probably :P